src/content/docs/codestream/observability/code-level-metrics.mdx

```diff
@@ -29,12 +29,20 @@ Click the CodeLens to see charts visualizing each of the metrics. If the reposit
 
 Along with a chart of the error rate you’ll also see a list of the actual errors happening in the same timeframe, including the number of occurrences for each. If the error rate is spiking, and you see one particular error is causing the problem, you can click on it to view the stack trace and [start collaborating](/docs/codestream/observability/error-investigation) on resolution.
 
-## Requirements [#requirements]
+## Coverage [#coverage]
 
-To see in-editor performance data, your service must meet the requirements listed below, and the service should have collected data in the last 30 minutes. The New Relic agent attempts to automatically collect data for functions on classes that are tied to HTTP requests. In many cases, and with many frameworks that use an MVC framework, these are often methods on a Controller class. 
+The New Relic agent attempts to automatically collect data for functions on classes that are tied to HTTP requests. In many cases, and with many frameworks that use an MVC framework, these are often methods on a Controller class. 
 
 Because all requests by the New Relic agent are not collected all the time, it’s possible that low-traffic methods won't see any data. If data is missing for a specific method that you wish to see results for, you can use custom instrumentation to fill any gaps. See guidance for [Java](/docs/apm/agents/java-agent/custom-instrumentation/java-custom-instrumentation), [.NET](/docs/apm/agents/net-agent/custom-instrumentation/introduction-net-custom-instrumentation), [PHP](/docs/apm/agents/php-agent/features/php-custom-instrumentation), [Python](/docs/apm/agents/python-agent/custom-instrumentation/python-custom-instrumentation), [Ruby](/docs/apm/agents/ruby-agent/api-guides/ruby-custom-instrumentation), [Go](/docs/apm/agents/go-agent/instrumentation/instrument-go-transactions) and [Node.js](/docs/apm/agents/nodejs-agent/extend-your-instrumentation/nodejs-custom-instrumentation).
 
+Run the following NRQL query to identify where you might see CodeLenses with code-level metrics for one of your services. Just replace the example `entity.guid` value with the one for the given service. In the query results, look at the `code.*` attributes to see if any of them represent your code, and not framework code.
+
+`select * from Span where entity.guid = 'MXxBUE18QVBQTElDQVRJT058MjM' where code.function is not null since 30 minutes ago LIMIT MAX`
```

Comment:
```suggestion
```
SELECT * 
FROM Span 
WHERE entity.guid = 'MXxBUE18QVBQTElDQVRJT058MjM' AND code.function is not null 
SINCE 30 minutes ago LIMIT MAX
```